app/tests/lib/helpers.js

```diff
@@ -73,19 +73,20 @@ define(function (require, exports, module) {
     done();
   }
 
-  function createRandomHexString(length) {
-    var str = '';
-    var lettersToChooseFrom = 'abcdefABCDEF01234567890';
-    var numberOfPossibilities = lettersToChooseFrom.length;
-
-    for (var i = 0; i < length; ++i) {
-      var indexToUse = Math.floor(Math.random() * numberOfPossibilities);
-      str += lettersToChooseFrom.charAt(indexToUse);
+  function createRandomString(length, base = 36) {
+    let str = '';
+    let alphabet = '0123456789abcdefghijklmnopqrstuvwxyz';
+
+    for (let i = 0; i < length; ++i) {
+      let indexToUse = Math.floor(Math.random() * base);
```

Comment:
> Why move the let outside the for? It seems like exactly what is designed for: block level scoping.

@jrgm thought he heard v8 has an issue where `let` inside of loops is significantly slower than `var`.